site.yml

```diff
@@ -74,15 +74,15 @@ asciidoc:
     owncloud-changelog-url: https://owncloud.com/changelog/server/
     php-supported-versions-url: https://secure.php.net/supported-versions.php
     http-status-codes-base-url: https://developer.mozilla.org/en-US/docs/Web/Status
-    minimum-php-version: 7.2
-    minimum-php-version-short-code: 72
+    minimum-php-version: 7.2.5
```

Comment:
This is used in some places where it generates a path to a file. For example: https://github.com/owncloud/docs/blob/master/modules/admin_manual/pages/configuration/server/caching_configuration.adoc

```
| PHP Version | Filename
| {minimum-php-version} | `/etc/php/{minimum-php-version}/mods-available/memcached.ini`
```
In the first column we might want to say `7.2.5` (or `7.2.5+` or?). In the 2nd column it needs to say only `7.2`

There might need to be more entries in `site.yml` that have different ways of expressing the version?

Reply:
Thanks for catching 👍 

There are two scenarious where `{minimum-php-version}` is used but in different ways.
1. as a print notice about the minimum level
2. as a variable for a path component

Having now a minimum patch release for PHP, my favorite is to keep the `{minimum-php-version}` variable and revert it to 7.2 and introduce a `{minimum-php-print}` variable having 7.2.5

Will take a closer look

Reply:
Yes, `minimum-php-version` is really `minimum-php-minor-version` - we could insert `minor` in all those sort of names, but then the var name is so long that it looks like a German word :)

Reply:
fixed, pls review again